scripts/start_gaudi_vllm_server.sh

```diff
@@ -149,9 +150,14 @@ case "$dtype" in
         echo Running with dtype="$dtype" ;;
     "fp8")
         echo Running with dtype="$dtype"
+	# TODO: quant config is different for different card number
```

Comment:
The config file is identical for different TP/EP sizes. Different TP/EP use different measured results.